pkg/api/server/register_images.go

```diff
@@ -30,21 +30,20 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	//   '200':
 	//     schema:
 	//     items:
-	//       $ref: "TBD"
+	//       $ref: "to be determined"
 	//   '404':
 	//     description: repo or image does not exist
 	//     schema:
-	//       $ref: "#/responses/generalError"
+	//       $ref: "#/responses/InternalError"
 	//   '500':
 	//     description: unexpected error
 	//     schema:
 	//      $ref: '#/responses/GenericError'
 	r.Handle(VersionedPath("/images/create"), APIHandler(s.Context, generic.CreateImageFromImage)).Methods(http.MethodPost).Queries("fromImage", "{fromImage}")
 	// swagger:operation POST /images/create images createImage
-	//
-	// Create an image from Source
-	//
 	// ---
+	// summary: Create an image from Source
```

Comment:
I have a hard time matching this endpoint to the API. https://docs.docker.com/engine/api/v1.40/#operation/ImageCreate has the `fromSrc` as a parameter.